src/controller/index.ts

```diff
@@ -142,10 +151,16 @@ export default class SimulariumController {
         } else if (netConnectionConfig) {
             const webSocketClient = new WebsocketClient(
                 netConnectionConfig,
+                this.useOctopus,
+                this.localBackendServer,
                 this.onError
             );
             this.remoteWebsocketClient = webSocketClient;
-            this.simulator = new RemoteSimulator(webSocketClient, this.onError);
+            this.simulator = new RemoteSimulator(
+                webSocketClient,
+                this.useOctopus,
```

Comment:
i wonder if useOctopus could be passed in to this createSimulatorConnection function (maybe even as part of NetConnectionParams) since it's not really needed for WebsocketClient, just for RemoteSimulator creation.

Reply:
I just moved useOctopus and secureConnection to NetConnectionParams, it looks cleaner and they are both inherently parameters related to the net connection. Thanks for the suggestion!